src/redux/categories/categories.js

```diff
@@ -1,7 +1,7 @@
-const CHECKSTATUS = 'CHECKSTATUS';
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As stated before, kindly make sure the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) as stated before.